cookbook/integration_openai_sdk.ipynb

```diff
@@ -14,14 +14,18 @@
     },
```

Comment:
<p>switch to decorators here as well for trace creation</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/langfuse/langfuse-docs/pull/415/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>